services/resource-coordinator/src/drivers/kubernetes/KubernetesDriver.js

```diff
@@ -0,0 +1,203 @@
+const { BaseDriver } = require('@openintegrationhub/resource-coordinator');
+const uuid = require('uuid/v4');
+const _ = require('lodash');
+const KubernetesRunningApp = require('./KubernetesRunningNode');
+const FlowSecret = require('./FlowSecret');
+
+class KubernetesDriver extends BaseDriver {
+    constructor(config, logger, k8s) {
+        super();
+        this._config = config;
+        this._logger = logger;
+        this._coreClient = k8s.getCoreClient();
+        this._batchClient = k8s.getBatchClient();
+    }
+
+    async createApp(flow, node, envVars, secretEnvVars) {
```

Comment:
What do you think about to implement smth like runCommand from e.io k8s driver instead of createApp? This solution looks cleaner and more generic: giving us better abstraction.